### PR TITLE
Implement wait-for command

### PR DESCRIPTION
**Pull request by pdalpra**
The `wait-for` effectively replace the usage of `timeout` + `nc` to wait for a container to be ready to use.

### CODE REVIEW

**Comment by pdalpra** [comment]
retest

**Comment by pdalpra** [comment]
retest

File: docker/wait_for_test.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+				So(err, ShouldBeNil)
+			})
+		})
+	})
+}
+
+func TestRetry(t *testing.T) {
+	Convey("Given the operation completes successfully before the timeout", t, func() {
+		f := func() error { return nil }
+		err := retry(f, time.Second, time.Duration(5)*time.Second)
```
Comment (PhilippeDupont) [inline docker/wait_for_test.go]: Why do you use `time.Duration(5)` instead of a simple `5` like you do in L122 for example or L60-62 ?

Comment (pdalpra) [inline docker/wait_for_test.go]: indeed, fixing it !

**Comment by pdalpra** [comment]
retest

**Comment by pdalpra** [comment]
retest

**Comment by pdalpra** [comment]
ping

**Comment by PhilippeDupont** [comment]
Checks have failed ? 
` 2016-04-08 21:09:54         It should complete without errors ✘✔✔✔ `

**Comment by pdalpra** [comment]
Ah, I now realize that I'm relying on localdocker being set, which is not the case on Jenkins :)

**Comment by pdalpra** [comment]
@gbougeard added in alias in test.sh/publish.sh from `./teads-central` to `teads-central`

**Comment by pdalpra** [comment]
cc @ebuzzing/platform for opinion on command format + documentation

**Comment by PhilippeDupont** [comment]
LGTM

**Pull request merged**